src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -352,17 +354,26 @@ private void validateCandidates() {
   }
 
   private void validateRules() {
-    if (getTiebreakMode() == Tabulator.TieBreakMode.MODE_UNKNOWN) {
+    if (getTiebreakMode() == TieBreakMode.MODE_UNKNOWN) {
       isValid = false;
       Logger.log(Level.SEVERE, "Invalid tie-break mode!");
     }
 
-    if (getOvervoteRule() == Tabulator.OvervoteRule.RULE_UNKNOWN) {
+    if ((getTiebreakMode() == TieBreakMode.RANDOM ||
+        getTiebreakMode() == TieBreakMode.PREVIOUS_ROUND_COUNTS_THEN_RANDOM ||
+        getTiebreakMode() == TieBreakMode.GENERATE_PERMUTATION) &&
+        getRandomSeed() == null) {
+      isValid = false;
+      Logger.log(Level.SEVERE, "When tiebreakMode involves a random element, randomSeed must be "
```

Comment:
If I set the tiebreak mode to "generatePermutation" and leave the seed box empty, it doesn't report this message for some reason (but it does with the other two). Instead I get:

> 2019-06-25 10:02:36 PDT SEVERE: Error processing candidate data:
> 2019-06-25 10:02:36 PDT SEVERE: Error during validation:
> java.lang.NullPointerException
> Validation failed!